### PR TITLE
Admin UI updates

### PR DESCRIPTION
**Pull request by tvdeyen**
The current admin UI was build for pre Solidus 2.0 times. It's time for an overhaul.

- Uses bootstrap grid classes
- Moves the admin views into the settings tabs
- Index table adjustments
- Translation fixes
- etcpp.

### CODE REVIEW

File: lib/solidus_social/engine.rb [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
-require 'solidus_social/facebook_omniauth_strategy_ext'
+require "omniauth-twitter"
+require "omniauth-facebook"
+require "omniauth-github"
+require "omniauth-google-oauth2"
+require "omniauth-amazon"
+require "deface"
+require "coffee_script"
+require "spree/core"
+require "solidus_social/facebook_omniauth_strategy_ext"
```
Comment (tvdeyen) [inline lib/solidus_social/engine.rb:11]: Sorry for the churn here. Forgot to disable autoformatting.

File: config/locales/en.yml [hunk 2]
```diff
@@ -7,13 +7,12 @@ en:
...
-      destroy: 'Successfully destroyed authentication method.'
-    back_to_authentication_methods_list: "Back To Authentication Methods List"
-    edit_social_method: "Editing Social Authentication Method"
+      destroy: "Successfully destroyed authentication method."
+    edit_social_method: "Edit %{provider}"
     environment: "Environment"
     new_social_method: "New Authentication Method"
     no_authentication_methods_found: "No Authentication Methods Found"
-    one_more_step: 'One more step to complete your registration from %{kind}'
+    one_more_step: "One more step to complete your registration from %{kind}"
```
Comment (tvdeyen) [inline config/locales/en.yml:15]: Argh autoformat stroke again. Bare with me

**Comment by kennyadsl** [comment]
Hey @tvdeyen, sorry I lost track of this. Can you please rebase when you have time? 🙏

**Comment by tvdeyen** [comment]
Already done in https://github.com/solidusio-contrib/solidus_social/pull/111

**Pull request closed without merging**